blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_1.py

```diff
@@ -0,0 +1,130 @@
+"""Formula 5.1 from NEN-EN 1992-1-1+C2:2011: Chapter Structural Analysis."""
+# pylint: disable=arguments-differ
+import numpy as np
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import M
+
+
+class Form5Dot1Imperfections(Formula):
+    """Class representing formula 5.1 for the calculation of initial inclination imperfections, Θi."""
```

Comment:
I think we should use some way to show it is $$\theta_i$$ and not $$\theta i$$

Not sure how Sphinx will handle it, but $$\theta i$$  isn't very clear, the subscript should be visible somehow.

Probably like we did before using math notation: :math:`\theta_i`.

Let's check that out before having to adjust it all manually.

Reply:
Sharp of you. I will check how this notations appear on the docs and change it to represent the real thing.

Reply:
<img width="557" alt="afbeelding" src="https://github.com/Blueprints-org/blueprints/assets/44614728/e0a246ba-3b44-4a3f-9006-aea8a6c173eb">

Not exactly what you would want to see. This needs adressing. We will get back to it separate of this PR.